staging/src/k8s.io/api/apps/v1beta1/types.go

```diff
@@ -58,6 +58,7 @@ type ScaleStatus struct {
 
 // +genclient=true
 // +noMethods=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
I think this change broke my PR #48746 which adds a new API version apps/v1beta2:

```
# k8s.io/kubernetes/vendor/k8s.io/api/apps/v1beta2
vendor/k8s.io/api/apps/v1beta2/register.go:47: cannot use Deployment literal (type *Deployment) as type runtime.Object in argument to scheme.AddKnownTypes:
        *Deployment does not implement runtime.Object (missing DeepCopyObject method)
...
```

I guess I can fix this error by adding this line to apps/v1beta2/types.go?

```go
// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Is it documented anywhere?